matter_server/common/helpers/util.py

```diff
@@ -35,7 +35,7 @@
 except:  # noqa
     # older python version
     NoneType = type(None)
-    UnionType = type(int | str)
```

Comment:
The reason this doesn't work with `from __future__` is because it is not written inside an annotation.

Reply:
Yeah, missed that while in the rush